src/rsz/src/Resizer.cc

```diff
@@ -309,6 +309,15 @@ bool Resizer::bufferBetweenPorts(Instance* buffer)
 bool Resizer::removeBuffer(Instance* buffer,
                            bool honorDontTouchFixed,
                            bool recordJournal)
+{
+  if (canRemoveBuffer(buffer, honorDontTouchFixed)) {
+    removeBuffer2(buffer, recordJournal);
```

Comment:
It'd be good to rename removeBuffer2 to something like doRemoveBuffer or removeBufferCore